client/webserver/site/src/js/locales.js

```diff
@@ -0,0 +1,121 @@
+export const ID_NO_PASS_ERROR_MSG = 'ID_NO_PASS_ERROR_MSG'
+export const ID_NO_APP_PASS_ERROR_MSG = 'ID_NO_APP_PASS_ERROR_MSG'
+export const ID_SET_BUTTON_BUY = 'ID_SET_BUTTON_BUY'
+export const ID_SET_BUTTON_SELL = 'ID_SET_BUTTON_SELL'
+export const ID_OFF = 'ID_OFF'
```

Comment:
This all seems peculiar to me.  Seems like these consts should just be an enumeration or something with integers.  Is this really what JavaScript forces us to do?

Reply:
Can leave this if it's Decrediton's approach, unless you or someone else has a better suggestion.